### PR TITLE
JFG-1065: Fix total throughput calculation

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/WorkloadAggregator.java [hunk 1]
```diff
@@ -136,16 +136,7 @@ private void aggregateValues(String sessionId, String taskId, WorkloadTask workl
...
-        if (Double.isNaN(throughput)) {
-            log.error("throughput is NaN (succeeded={},duration={}). Value for throughput will be set zero", succeeded, duration);
-            throughput = 0;
-        }
-        log.debug("Throughput: {}", throughput);
-
-        double successRate = (invoked.doubleValue() > 0)
-                ? Math.rint(succeeded / invoked.doubleValue() * 10000) / 10000
-                : 1;
+        double successRate = (invoked.doubleValue() > 0) ? Math.rint(succeeded / invoked.doubleValue() * 10000) / 10000 : 1;
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/WorkloadAggregator.java]: where is throughput now?

Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/WorkloadAggregator.java]: It's in DurationLogProcessor and it was here.
This code was just logging wrong throughput value, that's why it's deleted

Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/WorkloadAggregator.java]: please, review this code. looks like we don;t need latency and success rate as well

Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/WorkloadAggregator.java]: done

**Pull request merged**